lib/Sections/section3.dart

```diff
@@ -0,0 +1,155 @@
+        }
+
+        Widget _line2(username, data) {
+          return Align(
+            alignment: Alignment.bottomLeft,
+            child: Container(
+              padding: EdgeInsets.only(
+                left: 15.0,
+              ),
+              child: Wrap(
+                spacing: 20,
+                children: [
+                  Wrap(
+                    spacing: 5,
+                    children: [
+                      Icon(
+                        Icons.account_box,
+                        color: Color(0xFF868e96),
+                        size: 14,
+                      ),
+                      Text(
+                        username,
+                        style: TextStyle(
+                          color: Color(0xFF868e96),
+                          fontSize: 12,
+                        ),
+                      ),
+                    ],
+                  ),
+                  Text(
+                    data,
+                    style: TextStyle(
+                      color: Color(0xFF868e96),
+                      fontSize: 12,
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          );
+        }
+
+        return Column(children: [
+          _line1(newInfo['newTitle']),
+          _line2(newInfo['username'],newInfo['data']),
+        ]);
+      }
+
+      return Column(
+        children: news.map((e) => _news(e)).toList()
+      );
+    }
+
+    Widget _allDiscution() {
+      return Row(
+        children: [
+          Align(
+            alignment: Alignment.bottomLeft,
+            child: Container(
+              child: Text(
+                "Усі теми >",
+                style: TextStyle(
+                  color: Colors.white,
+                ),
+              ),
+              padding: EdgeInsets.only(
+                top: 10,
+                left: 15.0,
+                bottom: 15,
+              ),
+            ),
+          ),
+          Spacer(),
+          Container(
+              margin: EdgeInsets.all(10.0),
+              padding: EdgeInsets.only(left: 10, right: 10, top: 2, bottom: 2),
+              decoration: BoxDecoration(
+                  color: Colors.purple,
+                  borderRadius: BorderRadius.circular(
+                      3) // use instead of BorderRadius.all(Radius.circular(20))
+                  ),
+              // color: Colors.purple,
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Icon(
+                    Icons.manage_accounts,
+                    size: 16,
+                  ),
+                  Text(" Допомога перекладачам [ 567 ]"),
+                ],
+              ))
+        ],
+      );
+    }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        _titleSection3(),
+        _discutionsList(),
+        _allDiscution(),
+      ],
+    );
+  }
+}
```

Comment:
try to always end your file with empty line, some tools (especially linux-based) work better this way